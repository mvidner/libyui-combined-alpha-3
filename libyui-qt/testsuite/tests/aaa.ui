
# ---------------------------------------------------------
#
#  Filename:    aaa.ui
#
#  Purpose:     dummy test case for ui test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
#  Remarks:     there is more information on stderr
#
# ---------------------------------------------------------

{
    return( "After building the test environment new:"  +
            "all make clean, make -f Makefile.cvs, configure --prefix=/," +
            "make and make install as root." +
            "... there is more information on stderr ..." +

            "It's a libtool feature :-) When you make a program a script will be" + 
            "installed in the source directory which will do the final linking" +
            "just before you run it. (Just do more ui/Qt/qt to see what I mean)\n" +

            "The messages are warnings issued by the linker" +
            "(for some reason these warnings are enabled within SuSE linux). When" +
            "you run the program a second time, the program is already linked so you" +
            "won't get the warnings any more.\n" +

            "You can get rid of the warnings by removing the  \"-warn-common\" out" +
            "of your egcs specfile. No idea why it is in there in the first place." );
}

