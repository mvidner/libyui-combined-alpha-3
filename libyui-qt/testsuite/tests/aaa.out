(`stdio())
("After building the test environment new:all make clean, make -f Makefile.cvs, configure --prefix=/,make and make install as root.... there is more information on stderr ...It's a libtool feature :-) When you make a program a script will beinstalled in the source directory which will do the final linkingjust before you run it. (Just do more ui/Qt/qt to see what I mean)\nThe messages are warnings issued by the linker(for some reason these warnings are enabled within SuSE linux). Whenyou run the program a second time, the program is already linked so youwon't get the warnings any more.\nYou can get rid of the warnings by removing the  \"-warn-common\" outof your egcs specfile. No idea why it is in there in the first place.")
